cmd/qsctl/presign.go

```diff
@@ -41,6 +42,7 @@ func presignRun(_ *cobra.Command, args []string) (err error) {
 	}
 
 	t := task.NewReachFile(rootTask)
+	t.SetReacher(rootTask.GetStorage().(*qingstor.Storage))
```

Comment:
Why we need to convert to `*qingstor.Storage`? I think `storage.Reacher` is enough.